### ISSUE TITLE
deluge-console missing a dependency

### ISSUE THREAD

**Issue by PaperMountainStudio** [void-linux/void-packages#23382]
### System

* xuname:  
  *Void 5.7.6_1 x86_64 GenuineIntel uptodate rrrmDDDDDFFFFFFFFFFFFFFFFF*
* package:  
  *deluge-2.0.3_5*

### Expected behavior
deluge-console works without an error message

### Actual behavior
deluge-console works with an error message
```
% deluge-console --help
Unable to initialize gettext/locale!
'ngettext'
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/deluge/i18n/util.py", line 118, in setup_translation
    builtins.__dict__['_n'] = builtins.__dict__['ngettext']
KeyError: 'ngettext'
usage: deluge-console [-h] [-V] [-c <config>] [-l <logfile>] [-L <level>]
...
```

### Steps to reproduce
1. install deluge
2. run `deluge-console --help`

**Comment by PaperMountainStudio** [comment]
This is an upstream issue: https://dev.deluge-torrent.org/ticket/3342

**Comment by ericonr** [comment]
If you want to leave it open to track upstream, that's fine.

**Comment by PaperMountainStudio** [comment]
@ericonr ok, thanks, fixed in #23423
